src/Commands/DrupalCommands.php

```diff
@@ -88,11 +90,13 @@ public function siteInstall(array $options = [
       'account-name'      => InputOption::VALUE_REQUIRED,
       'account-password'  => InputOption::VALUE_REQUIRED,
       'account-mail'      => InputOption::VALUE_REQUIRED,
-      'database-user'     => InputOption::VALUE_REQUIRED,
-      'database-password' => InputOption::VALUE_REQUIRED,
+      'database-type'     => InputOption::VALUE_REQUIRED,
       'database-host'     => InputOption::VALUE_REQUIRED,
       'database-port'     => InputOption::VALUE_REQUIRED,
       'database-name'     => InputOption::VALUE_REQUIRED,
+      'database-user'     => InputOption::VALUE_REQUIRED,
+      'database-password' => InputOption::VALUE_REQUIRED,
```

Comment:
Only the following option is required:

* `database-type`

And these are optional:

* `database-host`
* `database-port`
* `database-user`
* `database-name`
* `database-password`

For example, SQLite doesn't specify the host, post, username and password, and some servers (like Oracle and MSSql) allow a dedicated server for a single database and you don't need to specify the database name.